Src/AutoMoq/AutoMockPropertiesCommand.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Moq;
+using Ploeh.AutoFixture.Kernel;
+
+namespace Ploeh.AutoFixture.AutoMoq
+{
+    /// <summary>
+    /// A command that populates all public writable properties/fields of a mock object with anonymous values.
+    /// </summary>
+    public class AutoMockPropertiesCommand : ISpecimenCommand
```

Comment:
After rethinking this, I think I may be over complicating things. And I agree that we should avoid introducing new types unless they're actually a good compelling reason to do so. 

I'll go ahead and redesign this bit as per the POC